adsmp/solr_updater.py

```diff
@@ -466,6 +464,9 @@ def transform_json_record(db_record):
                         db_record["bibcode"], type(links_data), links_data
                     )
                 )
+    if db_record.get("doctype_boost", None):
```

Comment:
I'm not sure whether this works. This was the only way I could think of where we can access the contents of BoostFactors DB and pass it to solr_updater. But this does not seem like good practice.